pyproject.toml

```diff
@@ -40,12 +38,10 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 
-requires-python = ">=3.7"
-dependencies = [
-    "pydantic>=1.10,<3",
-    "django>=3.1,<6",
-    "typing_extensions",
```

Comment:
Why do we remove `typing_extensions`?

Reply:
It's a good question. Since there wasn't a version specified and it's a [requirement](https://github.com/pydantic/pydantic/blob/b0dfc54c22372910638cd3854b6ed3872d8e457f/pyproject.toml#L49) of pydantic v2, I thought it would be fine to remove it.

Do you think I should put it back, and if so, what version? `>=4`?

Reply:
No, I don't think so because installing Pydantic v2 should install this library.